### ISSUE TITLE
Upnp 800 on playlist with google music

### ISSUE THREAD

**Issue by zoic21** [duncan3dc/sonos#35]
Hi,
I just swith from 1.3.5 to 1.6.1 and now I get upnp error 800 on all playlist using google music. I test with local music and it's ok.

I check track object and it's  : "duncan3dc\Sonos\Tracks\Deezer Object" , why I have deezer track object ? I don't use deezer...

I search and I found where is the issue, it's in Facttory.php in line 39 : 
```php
if (substr($uri, 0, strlen($class::PREFIX)) === $class::PREFIX) {
```
For google music prefix is the same as deezer : "x-sonos-http:"

I don't know how to fix that.....

Thank in advance

**Comment by duncan3dc** [comment]
Looks like the fallback to a basic Track doesn't work, I'll see what I can do

**Comment by duncan3dc** [comment]
This should be fixed in `1.7.0`

**Comment by zoic21** [comment]
Perfect it's work, thank a lot
